apidocs/src/check_permission.md

```diff
@@ -0,0 +1,50 @@
+## Check permission [/resources/{account}/{kind}/{identifier}?check=true&role={role}&privilege={privilege}]
+
+### Check permission [GET]
+
+Checks whether a role has a privilege on a resource. For example: is this Host
+authorized to `execute` (fetch the value of) this Secret?
+
+<!-- include(partials/resource_kinds.md) -->
+
+#### Example with `curl`
+
+Suppose your account name is "mycorp" and you want to check whether Host "application" can `execute` (fetch the value of) Variable "db-password":
+
+This request has a long URL, so I break it up for you with some variables.
+
+```bash
+endpoint='https://eval.conjur.org/resources'
+account='mycorp'
+var_id='db-password'
+host_id='application'
+
+curl -i -H "$(conjur authn authenticate -H)" \
+     '$endpoint/$account/variable/$var_id?check=true&role=$account:host:$host_id&privilege=execute'
+```
+
+---
+
+#### Request
+
+<!-- include(partials/auth_header_table.md) -->
+
+#### Response
+
+| Code | Description                                                                            |
+|------|----------------------------------------------------------------------------------------|
+|  204 | The role has the specified privilege on the resource                                   |
```

Comment:
Yeah, I thought we had other endpoints that were 200 but also an empty response. Perhaps not, I suppose there's usually data in the response come to think of it!